wordpress/wp-content/themes/MVMT/header.php

```diff
@@ -25,21 +25,75 @@
 	</head>
 	<body <?php body_class(); ?>>
 
+	<section class="NavTwo">
```

Comment:
In Section NavTwo, The spacing is inconsistent when compared to the Section Nav.